### ISSUE TITLE
Use new Telemetry SDK (Glean) in Android

### ISSUE THREAD

**Issue by linuxwolf** [mozilla-lockwise/lockwise-android#127]
Update the telemetry usage from the existing `service-telemetry` component to the new Telemetry SDK once available.

**Comment by linuxwolf** [comment]
Blocked waiting on the new SDK to be published

[ cc @irrationalagent to track ]

**Comment by eliserichards** [comment]
@irrationalagent is this still applicable or should we close this issue?

**Comment by irrationalagent** [comment]
I would say its still applicable but probably low priority

**Comment by linuxwolf** [comment]
cc @georgef for additional enablement/support once we're ready here.

**Comment by irrationalagent** [comment]
It is looking like lockwise may be a first customer of ecosystem telemetry. If that's the case, then the bit coming from the mobile apps will have to be implemented in glean.
